### PR TITLE
Use “BigInt values” to clarify they’re not objects

### LINKED ISSUE #3336: Issue with "BigInt": Adding n to a literal creates a bigint primitive, not a BigInt object
MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt

#### What information was incorrect, unhelpful, or incomplete?

This page reads:
> A BigInt is created by appending n to the end of an integer literal — 10n — or by calling the function BigInt().

where earlier a `BigInt` was defined to be an object:

> BigInt is a built-in object that provides a way to represent whole numbers...

This is, however, incorrect, as appending `n` to the end of an integer or calling the function `BigInt()` creates a `bigint` primitive, not a `BigInt` object.

Indeed, both `1n instanceof BigInt` and `BigInt(1) instanceof BigInt` will fail.

#### Specific section or headline?

BigInt > Description

First sentence

#### What did you expect to see?

I assume `BigInt` should have just been lowercase, as in:
> A bigint is created by appending n to the end of an integer literal — 10n — or by calling the function BigInt().

Although it may be a good idea to explicitly specify that it's a primitive that's being created.

Other MDN pages do correctly refer to the primitive by its lowercase name:

https://developer.mozilla.org/en-US/docs/Glossary/Primitive

> There are 7 primitive data types: string, number, bigint, boolean, undefined, symbol, and null.

#### Did you test this? If so, how?

Chrome dev console, running both:
```
1n instanceof BigInt
// Produces false
```
as well as
```
BigInt(1) instanceof BigInt
// Produces false
```

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/javascript/reference/global_objects/bigint`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/bigint/index.html
* Last commit: https://github.com/mdn/content/commit/3b80812a1d4f68e57f35fca763f50bf3e5dfe997
* Document last modified: 2021-02-19T19:57:02.000Z

</details>

### PR DESCRIPTION
**Pull request by sideshowbarker**
This change updates the BigInt article to clarify that code does not operate on BigInt objects but instead on bigint primitive values. The ECMAScript spec uses the term “BigInt value” to refer to bigint primitives — and the shorthand “BigInt” to refer to BigInt values — and normatively defines what a BigInt value is:

https://tc39.es/ecma262/#sec-terms-and-definitions-bigint-value
> ### 4.4.28 BigInt value
> primitive value corresponding to an arbitrary-precision integer value

…so this change follows that, and updates the BigInt article to consistently use “BigInt value”, in normal type — non-monospaced, without `<code></code>` markup — to make clear that what’s being operated on in code is not a BigInt *object*, but instead a primitive value.

Fixes https://github.com/mdn/content/issues/3336

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/bigint/index.html [hunk 1]
```diff
@@ -33,26 +33,28 @@ <h2 id="Description">Description</h2>
 // ↪ 9007199254740991n
 </pre>
 
-<p><code>BigInt</code> is similar to <a href="/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number"><code>Number</code></a> in some ways, but also differs in a few key matters — it cannot be used with methods in the built-in <a href="/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math"><code>Math</code></a> object and cannot be mixed with instances of <code>Number</code> in operations; they must be coerced to the same type. Be careful coercing values back and forth, however, as the precision of a <code>BigInt</code> may be lost when it is coerced to a <code>Number</code>.</p>
+<p>BigInt values are similar to Number values in some ways, but also differ in a few key matters: a BigInt value cannot be used with methods in the built-in <a href="/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math"><code>Math</code></a> object and cannot be mixed with a Number value in operations; they must be coerced to the same type. Be careful coercing values back and forth, however, as the precision of a BigInt value may be lost when it is coerced to a Number value.</p>
```
Comment (Rumyra) [inline files/en-us/web/javascript/reference/global_objects/bigint/index.html:36]:
```suggestion
<p>BigInt values are similar to Number values in some ways, but also differ in a few key matters: A BigInt value cannot be used with methods in the built-in <a href="/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math"><code>Math</code></a> object and cannot be mixed with a Number value in operations; they must be coerced to the same type. Be careful coercing values back and forth, however, as the precision of a BigInt value may be lost when it is coerced to a Number value.</p>
```

File: files/en-us/web/javascript/reference/global_objects/bigint/index.html [hunk 2]
```diff
@@ -33,26 +33,28 @@ <h2 id="Description">Description</h2>
...
-<p>When wrapped in an <code>Object</code>, a <code>BigInt</code> will be considered as a normal "object" type:</p>
+<p>A BigInt value can also be wrapped in an <code>Object</code>:</p>
 
 <pre class="brush: js">typeof Object(1n) === 'object'  // true
 </pre>
 
 <h3 id="Operators">Operators</h3>
 
-<p>The following operators may be used with <code>BigInt</code>s (or object-wrapped <code>BigInt</code>s): <code>+</code>, <code>*</code>, <code>-</code>, <code>**</code>, <code>%</code>.</p>
+<p>The following operators may be used with a BigInt values or object-wrapped BigInt values:</p>
```
Comment (Rumyra) [inline files/en-us/web/javascript/reference/global_objects/bigint/index.html:53]:
```suggestion
<p>The following operators may be used with BigInt values or object-wrapped BigInt values:</p>
```

**Review by Rumyra — APPROVED** [review]
Honestly the smallest nits - approving so you can merge when you need 👍

**Comment by sideshowbarker** [comment]
> Honestly the smallest nits - approving so you can merge when you need 👍

Thanks much for catching those

**Pull request merged**